markdown/6-Constraint-Satisfaction-Problems/exercises/ex_6/answer.md

```diff
@@ -1 +1,10 @@
-Not Available
\ No newline at end of file
+Let `X = (A, B).`
+
+The ternary constraint `A + B = C` can be expressed in terms of the following 3 binary constraints:
+1. `A = X1`
+2. `B = X2`
```

Comment:
The question expects a single auxiliary variable for ternary to binary constraint transformation in CSPs. 
So although your answer is correct, the inclusion of X2 brings the second auxiliary variable (first is X1).

Reply:
Thanks for reviewing so quickly!

Would the answer instead be to set `X` to a 3-tuple, each value being a value in the domain of A, B, C?